localization_launch/launch/util/util.launch.py

```diff
@@ -106,30 +116,13 @@ def add_launch_arg(name: str, default_value=None, description=None):
         "path to the parameter file of random_downsample_filter",
     )
     add_launch_arg("use_intra_process", "true", "use ROS2 component container communication")
+    add_launch_arg("use_pointcloud_container", "False")
+    add_launch_arg("container_name", "perception_pipeline_container")
```

Comment:
why `perception_pipeline_container`?
I think `pointcloud_container` is better.

Reply:
Sorry, It is my mistake...